flex/06-flex-layout/README.md

```diff
@@ -4,6 +4,7 @@ Flexbox is useful for laying out entire pages as well as the smaller components
 
 ### Hints
 - You may want to search something like `CSS remove list bullets`.  We've done this for you in previous examples, but not here. Yay learning.
+- Finding out how to style links in CSS might help you get rid of that pesky underscore...
```

Comment:
I can definitely see this hint helping people, especially since this is still fairly early on. I think the only thing I would change is the "underscore" to "underline":
```suggestion
- Finding out how to style links in CSS might help you get rid of that pesky underline decoration...
```

Reply:
I am not sure whether I have to comment if I did commit the suggestion but here I am, confirming that this absolutely makes sense.

Thank you!

Reply:
Thank you for commenting! In the future (and this should apply to all of our repos I believe), after making requested changes you should be able to press the "re-request review" button in the Reviewers section at the top of the right side bar. In this case a comment worked because I got an email either way 😆